Content.Shared/_RMC14/Xenonids/Stomp/SharedXenoStompSystem.cs

```diff
@@ -52,7 +53,9 @@ private void OnXenoStompAction(Entity<XenoStompComponent> xeno, ref XenoStompAct
             return;
 
         _receivers.Clear();
+        _shortReceivers.Clear();
         _entityLookup.GetEntitiesInRange(xform.Coordinates, xeno.Comp.Range, _receivers);
+        _entityLookup.GetEntitiesInRange(xform.Coordinates, xeno.Comp.ShortRange, _shortReceivers);
```

Comment:
You should be able to get the transform component when looping over receivers and then get the distance, instead of doing two lookups